1_1_vulns/it/LLM10_ModelTheft.md

```diff
@@ -1,49 +1,49 @@
-4. Regularly monitor and audit access logs and activities related to LLM model repositories to detect and respond to any suspicious or unauthorized behavior promptly.
-5. Automate MLOps deployment with governance and tracking and approval workflows to tighten access and deployment controls within the infrastructure.
-6. Implement controls and mitigation strategies to mitigate and|or reduce risk of prompt injection techniques causing side-channel attacks.
-7. Rate Limiting of API calls where applicable and|or filters to reduce risk of data exfiltration from the LLM applications, or implement techniques to detect (E.G., DLP) extraction activity from other monitoring systems.
-8. Implement adversarial robustness training to help detect extraction queries and tighten physical security measures.
-9. Implement a watermarking framework into the embedding and detection stages of an LLMs lifecyle.
+1. Implementare controlli di accesso robusti (ad esempio, RBAC e minimizzazopne dei privilegi) e meccanismi di autenticazione forti per limitare l'accesso non autorizzato ai repository di modelli LLM e agli ambienti di arricchimento e input (pre-training e training).
+   1. Questo è particolarmente vero per i primi tre casi che più comunemente potrebbero causare questa vulnerabilità quali minacce interne (insider threat), configurazione errata e/o controlli di sicurezza deboli, relativi all'infrastruttura che ospita modelli, pesi e architettura dell'LLM, in cui un attore malintenzionato potrebbe infiltrarsi dall'interno, o dall'esterno.
+   2. La gestione dei fornitori, il controllo, la verifica e il controllo del livello di dipendenza da essi sono elementi importanti di attenzione per prevenire gli abusi ed attacchi diretti sulla catena di fornitura.
+2. Limitare l'accesso dell'LLM alle risorse di rete, ai servizi interni e alle API.
+   1. Questa pratica è efficace per tutti gli esempi di attacchi comuni, fornendo copertura dai rischi e dalle minacce interne (insider threats), e limitando allo stesso tempo ciò a cui l'applicazione LLM "_ha accesso_", rappresentando dunque un meccanismo o metodo preventivo, per prevenire o interrompere attacchi laterali (side-channel attacks).
+3. Utilizzare un Inventario o Registro centralizzato per i modelli di Machine Learning (ML Models) utilizzati in produzione. Avere un registro di modelli centralizzato impedisce l'accesso non autorizzato ai modelli di ML tramite controlli di accesso, autenticazione e grazie alla capacità di monitoraggio/registrazione, elemnti fondamentali per il governo di questi processi (governance). Avere un repository centralizzato è inoltre vantaggioso per collezionare dati sugli algoritmi utilizzati dai modelli, ai fini della conformità, valutazione dei rischi e loro mitigazione.
```

Comment:
```suggestion
3. Utilizzare un Inventario o Registro centralizzato per i modelli di Machine Learning utilizzati in produzione. Avere un registro di modelli centralizzato impedisce l'accesso non autorizzato ai modelli di ML tramite controlli di accesso, autenticazione e grazie alla capacità di monitoraggio/registrazione, elementi fondamentali per la governance di questi processi. Avere un repository centralizzato è inoltre vantaggioso per raccogliere dati sugli algoritmi utilizzati dai modelli ai fini di conformità, valutazione dei rischi e loro mitigazione.
```

Reply:
I tried to follow the rule of "not using anglophone terms without at least a translation next to them".